.gitignore

```diff
@@ -17,3 +17,4 @@ coverage
 Procfile
 .lankrc*
 test-output
+package-lock.json
```

Comment:
Do we ignore `package-lock.json` generally? It definitely is hard to keep both this and `yarn.lock` in sync

Reply:
Yeah -- we're doing "yarn only" :P

Reply:
No complaints from me 😆